syncer/ddl_test.go

```diff
@@ -373,22 +349,25 @@ func (s *testSyncerSuite) TestResolveGeneratedColumnSQL(c *C) {
 		ast1, err := parser.ParseOneStmt(tc.sql, "", "")
 		c.Assert(err, IsNil)
 
-		sqls, _, _, err := syncer.resolveDDLSQL(tc.sql, parser, "")
+		sqls, _, err := syncer.resolveDDLSQL(parser, ast1, "test")
 		c.Assert(err, IsNil)
 
 		c.Assert(len(sqls), Equals, 1)
 		getSQL := sqls[0]
 		c.Assert(getSQL, Equals, tc.expected)
 
-		ast2, err := parser.ParseOneStmt(getSQL, "", "")
-		c.Assert(err, IsNil)
+		// FIXME: remove me after implement generated function in restore
```

Comment:
latest parser has fixed this bug, we can remove these comments now

Reply:
it's useless, because of some very tricky code,  I decide remove these code